src/category_theory/discrete_category.lean

```diff
@@ -52,47 +61,64 @@ end functor
 
 namespace nat_trans
 
-def of_homs {I : Type u₁} {F G : discrete I ⥤ C}
+/--
+For functors out of a discrete category,
+a natural transformation is just a collection of maps,
+as the naturality squares are trivial.
+-/
+def of_function {I : Type u₁} {F G : discrete I ⥤ C}
```

Comment:
`of_functions`?

Reply:
I agree the naming here is not good. I wanted uniformity across the family constructing functors/transformations/isomorphisms/equivalences.

Perhaps I should put all of them in the `discrete` namespace, as `discrete.functor`, `discrete.nat_trans`, `discrete.nat_iso` and `discrete.equivalence`?

Reply:
That sounds good to me!

Reply:
Although it might be nice to distinguish between constructing a functor to a discrete category and constructing a functor from a discrete category, I feel like it might be helpful to have a constructor for the former too.

Reply:
Required a bit of search and replace, but this now looks much nicer!